examples/sample-app/application-template-custombuild.json

```diff
@@ -145,6 +145,9 @@
             "name": "origin-ruby-sample:latest"
           }
         },
+        "postCommit": {
+          "script": "bundle exec rake test"
+        },
```

Comment:
yeah, unless we're going to update the origin-custom-docker-builder image to actually respect/invoke the hook, we probably shouldn't define it since it implies behavior that isn't present.